src/imitation/data/rollout.py

```diff
@@ -345,8 +345,11 @@ def rollout_stats(trajectories: Sequence[types.TrajectoryWithRew]) -> dict:
     stat_names = ["min", "mean", "std", "max"]
     for desc_name, desc_vals in traj_descriptors.items():
         for stat_name in stat_names:
-            stat_value = getattr(np, stat_name)(desc_vals)
-            out_stats[f"{desc_name}_{stat_name}"] = stat_value
+            stat_value: np.generic = getattr(np, stat_name)(desc_vals)
+            out_stats[f"{desc_name}_{stat_name}"] = stat_value.item()
+
+    for v in out_stats.values():
+        assert isinstance(v, (int, float))
```

Comment:
These return values need to be non-numpy because otherwise they don't return properly as run-results from expert_demos_ex, etc.

I could add this add as a comment to the code itself if that seems helpful.

Reply:
Hmm, it might be worth mentioning briefly that the NumPy is returning a NumPy type even for scalars, but I don't think we need to mention the scripts since this is in a non-script file.

Reply:
Added a comment to that effect.